sdk/ai/azure-ai-inference/README.md

```diff
@@ -451,6 +459,89 @@ TBD
+
+### Setup
+
+The environment variable AZURE_TRACING_GEN_AI_CONTENT_RECORDING_ENABLED controls whether the actual message contents will be included in the traces or not. By default, the message contents are not include as part of the trace. Set the value of the environment variable to true (case insensitive) for the message contents to be included as part of the trace. Any other value will cause the message contents not to be traced.
+
+You also need to configure the tracing implementation in your code, like so:
+
+```
+from azure.core.settings import settings
+settings.tracing_implementation = "opentelemetry"
+```
+
+### Trace Exporter(s)
+
+In order for the traces to be captured, you need to setup the applicable trace exporters. The chosen exporter will be based on where you want the traces to be output. You can also implement your own exporter. The first example below shows how to setup an exporter to Azure Monitor.
+Please refer to [this](https://learn.microsoft.com/en-us/azure/azure-monitor/app/create-workspace-resource?tabs=bicep) documentation for more information about how to created Azure Monitor resource.
+Configure the APPLICATIONINSIGHTS_CONNECTION_STRING based on your Azure Monitor resource.
+
+```
+# Setup tracing to Azure Monitor
+from azure.monitor.opentelemetry.exporter import AzureMonitorLogExporter
+trace.set_tracer_provider(TracerProvider())
+tracer = trace.get_tracer(__name__)
+span_processor = BatchSpanProcessor(
+    AzureMonitorTraceExporter.from_connection_string(
+        os.environ["APPLICATIONINSIGHTS_CONNECTION_STRING"]
+    )
+)
+trace.get_tracer_provider().add_span_processor(span_processor)
+```
+
+The following example shows how to setup tracing to console output.
+
+```
+# Setup tracing to console
+exporter = ConsoleSpanExporter()
+trace.set_tracer_provider(TracerProvider())
+tracer = trace.get_tracer(__name__)
+trace.get_tracer_provider().add_span_processor(SimpleSpanProcessor(exporter))
+```
+### Instrumentation
+
+Use the AIInferenceInstrumentor to instrument the Azure AI Inferencing API for LLM tracing, this will cause the LLM traces to be emitted from Azure AI Inferencing API.
+
+```
+from azure.core.tracing import AIInferenceApiInstrumentor
+# Instrument AI Inference API
+AIInferenceApiInstrumentor().instrument()
+```
+
+It is also possible to uninstrument the Azure AI Inferencing API by using the uninstrument call. After this call, the LLM traces will no longer be emitted by the Azure AI Inferencing API until instrument is called again.
+
+```
+AIInferenceApiInstrumentor().uninstrument()
+```
+
+### Tracing Your Own Functions
+The @tracer.start_as_current_span decorator can be used to trace your own functions. This will trace the function parameters and their values. You can also add further attributes to the span in the function implementation as demonstrated below. Note that you will have to setup the tracer in your code before using the decorator.
```

Comment:
Please add a link to https://opentelemetry.io/docs/languages/python/ so that users know where to find more info

Reply:
added